kie-remote/kie-remote-jaxb-gen/pom.xml

```diff
@@ -292,11 +268,10 @@
               <goal>schemagen</goal>
             </goals>
             <configuration>
-              <includes>
-                <include>org/drools/**/*.java</include>
-                <include>org/jbpm/**/*.java</include>
-              </includes>
-              <verbose>true</verbose>
+              <sources>
+                <source>${project.basedir}/src/main/java/org/drools</source>
+                <source>${project.basedir}/src/main/java/org/jbpm</source>
+              </sources>
```

Comment:
I've tried that, but could not make it work. The `source` expect either directory or specific file path. Using GLOBs does not seem to be supported.